src/app/components/layout-sidebar/layout-sidebar-demo.component.html

```diff
@@ -1,3 +1,3 @@
-<stache-layout-sidebar>
+<stache layout="sidebar">
```

Comment:
Changed this so that consumers can no longer use `<stache-layout-sidebar>`. They must use either `<stache layout="sidebar">` or `<stache-layout layoutType="sidebar">`. Eases confusion...